politeiawww/api/v1/v1.go

```diff
@@ -37,6 +37,7 @@ const (
 	RouteLogin                  = "/login"
 	RouteLogout                 = "/logout"
 	RouteSecret                 = "/secret"
+	RouteUserProposalAccessTime = "/user/proposals/{token:[A-z0-9]{64}}/accesstime"
```

Comment:
`/user/proposals` refers to the proposals that have been submitted by the user, so having `accesstime` as a sub-route of the user's proposal collection was a bit confusing since you're using it to get the access times of all proposals, not just the user's proposals.  

Another thing that I found confusing is that you use this route as a GET request to return proposal access times for all proposals, but the route includes a specific proposal resource in the path.

You're also using a `POST` request, but the call is idempotent.  I would use a `PUT` request.

The way that I would do it would be to have the endpoints:
`GET /user/accesstimes`
`PUT /user/accesstimes/{token}`

The `GET` request is used to fetch the accesstimes for all vetted proposals and the `PUT` request is used to update the access time for a specific proposal.

Reply:
I misunderstood part of this.  I thought we needed to return access times for all proposals, but we only need to return the access time for one proposal at a time.  The routes I would use would be:

`GET /user/accesstime/{token}`
`PUT /user/accesstime/{token}`

Reply:
After discussing it on slack, we agreed that a route like `GET /user/accesstimes/{token}` to fetch the access time for a specific proposal makes more sense because we won't be using the access times for all proposals at the same time.